nengo_loihi/decode_neurons.py

```diff
@@ -19,6 +20,7 @@ class DecodeNeurons:
     ----------
     dt : float
         Time step used by the simulator.
+    is_input : bool, optional (Default: False)
```

Comment:
Should this be here? Doesn't look like the constructor accepts this.

Reply:
No, I had this here for a while then moved it to a subclass. Will fix.

Reply:
Done in fe055cee